### PR TITLE
Update Azure roles instructions to include the necessary permissions

### PR DESCRIPTION
**Pull request by boaz0**
closes JIRA ticket 53

//cc @nlcwong @dayleparker @rvsia 

image
image
image

### CODE REVIEW

File: packages/sources/src/addSourceWizard/hardcodedSchemas.js [hunk 1]
```diff
@@ -204,21 +204,26 @@ export default {
...
+                        name: 'azure-sub-id',
+                        nextStep: 'configure-roles',
+                      fields: [{
+                        name: 'azure-sub-id-description',
+                        component: 'description',
+                        Content: CMAzure.SubscriptionID,
+                      }, {
+                        name: 'credentials.subscription_id',
+                        component: componentTypes.TEXT_FIELD,
+                        label: 'Subscription ID'
```
Comment (rvsia) [inline packages/sources/src/addSourceWizard/hardcodedSchemas.js:217]: Please move this label to 'credentials.subscription_id' object one level up - those objects are used for getting info in the summary.

Comment (boaz0) [reply packages/sources/src/addSourceWizard/hardcodedSchemas.js:217]: I thought I did :) in line 129.

**Comment by boaz0** [comment]
@nlcwong - we are waiting on your approval. Please take a look. Thanks :1st_place_medal:

**Comment by nlcwong** [comment]
@boaz0 In step 4, remove the word Azure from both the wizard nav and step title. That way we are consistent with all the other step titles (without using the term Azure in there). I was inconsistent in my mocks, sorry about that.

Otherwise, LGTM!

**Comment by boaz0** [comment]
@nlcwong done!

**Pull request merged**